# Repo_skeleton

**Repo_skeleton** contains default notebooks (project documenattion file structure) to copy over to a new repo.

## Notebook Templates

Hereâ€™s a categorised list of common bioinformatics analysis pipelines, for which we provide template repos.

### Genomics

| *made* | *template notebook* | *description* |
|--------|---------------------|---------------|  
| [ x ] | **notebook_genome_assembly/**   | de novo genome assembly from long or short reads  
| [  ] | **notebook_genome_annotation/**  | structural and functional genome annotation  
| [  ] | **notebook_wgs/**                | whole genome resequencing analysis pipeline  
| [  ] | **notebook_wes/**                | whole exome sequencing pipeline  
| [  ] | **notebook_pan_genome/**         | construction and analysis of pan-genomes from multiple genomes  
| [  ] | **notebook_structural_variant/** | detection and analysis of large structural variants and CNVs (copy number variants)  


### Transcriptomics

| *made* | *template notebook* | *description* |
|--------|---------------------|---------------| 
[ x ] | **notebook_bulk_rnaseq/**            | differential expression analysis of bulk RNA-Seq data  
[  ] | **notebook_total_rnaseq/**           | total rnaseq including non-coding and ribo-depleted data  
[  ] | **notebook_small_rnaseq/**           | analysis of small RNAs such as miRNAs and piRNA  
[  ] | **notebook_singlecell_rnaseq/**      | (single-cell) scRNA-seq clustering, annotation and visualization  
[  ] | **notebook_transcriptome_assembly/** | de novo transcript reconstruction from RNA-Seq reads  
[  ] | **notebook_alternative_splicing/**   | detection and quantification of alternative splicing events  


### Epigenomics & Regulatory

| *made* | *template notebook* | *description* |
|--------|---------------------|---------------| 
[  ] | **notebook_chip_seq/**             | transcription factor or histone ChIP-seq analysis  
[  ] | **notebook_atac_seq/**             | analysis of chromatin accessibility using ATAC-seq  
[  ] | **notebook_methylation_seq/**      | bisulfite sequencing or array-based dna methylation analysis  
[  ] | **notebook_hic/**                  | genome-wide chromatin interaction analysis using Hi-C  
[  ] | **notebook_regulatory_landscape/** | integration of ChIP, ATAC and RNA-Seq for regulatory element mapping  


### Microbiome & Metagenomics

| *made* | *template notebook* | *description* |
|--------|---------------------|---------------| 
[  ] | **notebook_16s/**                  | amplicon-based microbiome profiling using 16S rRNA  
[  ] | **notebook_metagenomics/**         | shotgun metagenome analysis including taxonomy and function  
[  ] | **notebook_metatranscriptomics/**  | expression profiling of microbial communities from RNA  
[  ] | **notebook_mag_reconstruction/**   | binning and annotation of metagenome-assembled genomes  


### Functional & Comparative

| *made* | *template notebook* | *description* |
|--------|---------------------|---------------| 
[  ] | **notebook_functional_enrichment/** | GO, KEGG or Pfam-based pathway and functional enrichment  
[  ] | **notebook_synteny/**               | comparative genome alignment and synteny block analysis  
[  ] | **notebook_phylogenomics/**         | phylogenetic tree construction from genome-scale data  


### Proteomics

| *made* | *template notebook* | *description* |
|--------|---------------------|---------------| 
[  ] | **notebook_proteomics/**          | identification and quantification of proteins from LC-MS/MS data  
[  ] | **notebook_phosphoproteomics/**   | analysis of post-translational modifications (PTMs) like phosphorylation  


### Specialized & Advanced

| *made* | *template notebook* | *description* |
|--------|---------------------|---------------| 
[  ] | **notebook_crispr_screen/**       | analysis of CRISPR pooled screens and gene knockout studies  
[  ] | **notebook_clinical_genomics/**   | variant interpretation using clinical and pathogenicity guidelines  
[  ] | **notebook_qtl_mapping/**         | discovery of eQTLs, sQTLs or methylation QTLs  
[  ] | **notebook_bulk_deconvolution/**  | estimation of cell-type proportions from bulk RNA-Seq  
[  ] | **notebook_multi_omics/**         | integration of transcriptomic, epigenomic and proteomic data   



